packages/@textlint/markdown-to-ast/test/fixtures/footnote-inline.text/output.json

```diff
@@ -6,121 +6,80 @@
             "children": [
                 {
                     "type": "Str",
-                    "value": "This is an example of an inline footnote.",
+                    "value": "This is an example of an inline footnote.[^This is the ",
```

Comment:
- [ ] it requires https://github.com/remarkjs/remark-footnotes#optionsinlinenotes ?